intrahospital_api/deployment_check.py

```diff
@@ -0,0 +1,201 @@
+"""
+Gets called by the deployment check management command.
+
+check_since is the main method
+
+Takes a datetime and looks at all patients who don't have archived tags.
+
+Looks at all lab tests that have a last_updated after the datetime for those patients.
```

Comment:
Can we make explicit what 'looks at' means - the phrase is used 3 times in 3 lines, but I still don't know what this 'looks at' or what question about these things we answer by calling `check_since()`.